### ISSUE TITLE
user should be able to disable posting for testing purposes

### ISSUE THREAD

**Issue by thagler1** [henkhaus/reddit_bot_testing#7]
since it takes 10 minutes due to rate limiting. send all posts to the output stream instead. We could add a boolean value in the secrets.py file the decides if it posts or prints.

**Comment by henkhaus** [comment]
I agree, we could potentially put that switch when the bot is initialized: ie RedditBot('example', dev/prod)
